peribolos.yaml

```diff
@@ -28,3 +28,7 @@ orgs:
         default_branch: main
         description: UOR sample Python gRPC client
         has_projects: true
+      ai-model-registry:
```

Comment:
Can this be added as to preserve alphabetic order?

Reply:
Fixed